pkg/cloudprovider/const.go

```diff
@@ -0,0 +1,23 @@
+package cloudprovider
+
+const (
+	// IBM Cloud Service Name constants.
```

Comment:
given @JoelSpeed 's comment, i think it's worthwhile to add a comment here explaining what these are used for/by. eg something 

```
// These constants are used by MAO, MCO, CIO, Installer to identify blah blah blah
```

Reply:
Sure, will do

Reply:
Updated